gwswlib/sql_models/constants.py

```diff
@@ -0,0 +1,226 @@
+def reversed_dict(d):
+    """Create a reverse lookup dictionary"""
```

Comment:
create => return

En even zeggen wat `d` is. Het klinkt als `dict`, maar dan klopt de list comprehension niet.